Redundant std::move in json_parsing.cpp

I have problem with the compilation of cppreststdk 2.10.15 with gcc 9.3.0. See errror:

```
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp: In instantiation of 'std::unique_ptr<web::json::details::_Value> web::json::details::JSON_Parser<CharType>::_ParseValue(web::json::details::JSON_Parser<CharType>::Token&) [with CharType = char]':
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:131:33:   required from 'web::json::value web::json::details::JSON_Parser<CharType>::ParseValue(web::json::details::JSON_Parser<CharType>::Token&) [with CharType = char]'
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1134:39:   required from here
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1077:35: error: redundant move in return statement [-Werror=redundant-move]
 1077 |             return std::move(value);
      |                                   ^
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1077:35: note: remove 'std::move' call
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1089:35: error: redundant move in return statement [-Werror=redundant-move]
 1089 |             return std::move(value);
      |                                   ^
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1089:35: note: remove 'std::move' call
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1096:35: error: redundant move in return statement [-Werror=redundant-move]
 1096 |             return std::move(value);
      |                                   ^
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1096:35: note: remove 'std::move' call
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1103:35: error: redundant move in return statement [-Werror=redundant-move]
 1103 |             return std::move(value);
      |                                   ^
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1103:35: note: remove 'std::move' call
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp: In instantiation of 'std::unique_ptr<web::json::details::_Value> web::json::details::JSON_Parser<CharType>::_ParseArray(web::json::details::JSON_Parser<CharType>::Token&) [with CharType = char]':
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1069:20:   required from 'std::unique_ptr<web::json::details::_Value> web::json::details::JSON_Parser<CharType>::_ParseValue(web::json::details::JSON_Parser<CharType>::Token&) [with CharType = char]'
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:131:33:   required from 'web::json::value web::json::details::JSON_Parser<CharType>::ParseValue(web::json::details::JSON_Parser<CharType>::Token&) [with CharType = char]'
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1134:39:   required from here
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1043:44: error: redundant move in return statement [-Werror=redundant-move]
 1043 |                     return std::move(result);
      |                                            ^
/azure-rest-sdk/cpprestsdk/Release/src/json/json_parsing.cpp:1043:44: note: remove 'std::move' call
[  8%] Building CXX object Release/src/CMakeFiles/cpprest.dir/json/json_serialization.cpp.o
cc1plus: all warnings being treated as errors
make[2]: *** [Release/src/CMakeFiles/cpprest.dir/build.make:206: Release/src/CMakeFiles/cpprest.dir/json/json_parsing.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:115: Release/src/CMakeFiles/cpprest.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```

Its look like problem on gcc 9.3.0.  Can I fix it with disabled warning -DWERROR=0  ?

--- a/Release/src/json/json_parsing.cpp
+++ b/Release/src/json/json_parsing.cpp
@@ -1040,7 +1040,7 @@ std::unique_ptr<web::json::details::_Value> JSON_Parser<CharType>::_ParseArray(
                 case JSON_Parser<CharType>::Token::TKN_CloseBracket:
                     GetNextToken(tkn);
                     if (tkn.m_error) return utility::details::make_unique<web::json::details::_Null>();
-                    return std::move(result);
+                    return result;
                 default:
                     SetErrorCode(tkn, json_error::malformed_array_literal);
                     return utility::details::make_unique<web::json::details::_Null>();
@@ -1074,7 +1074,7 @@ std::unique_ptr<web::json::details::_Value> JSON_Parser<CharType>::_ParseValue(
                                                                                     tkn.has_unescape_symbol);
             GetNextToken(tkn);
             if (tkn.m_error) return utility::details::make_unique<web::json::details::_Null>();
-            return std::move(value);
+            return value;
         }
         case JSON_Parser<CharType>::Token::TKN_IntegerLiteral:
         {
@@ -1086,21 +1086,21 @@ std::unique_ptr<web::json::details::_Value> JSON_Parser<CharType>::_ParseValue(
 
             GetNextToken(tkn);
             if (tkn.m_error) return utility::details::make_unique<web::json::details::_Null>();
-            return std::move(value);
+            return value;
         }
         case JSON_Parser<CharType>::Token::TKN_NumberLiteral:
         {
             auto value = utility::details::make_unique<web::json::details::_Number>(tkn.double_val);
             GetNextToken(tkn);
             if (tkn.m_error) return utility::details::make_unique<web::json::details::_Null>();
-            return std::move(value);
+            return value;
         }
         case JSON_Parser<CharType>::Token::TKN_BooleanLiteral:
         {
             auto value = utility::details::make_unique<web::json::details::_Boolean>(tkn.boolean_val);
             GetNextToken(tkn);
             if (tkn.m_error) return utility::details::make_unique<web::json::details::_Null>();
-            return std::move(value);
+            return value;
         }
         case JSON_Parser<CharType>::Token::TKN_NullLiteral:
         {

